assignment1.txt

```diff
@@ -0,0 +1,16 @@
+1. Compiler is a program which translates codes written 
```

Comment:
right, more specifically, a compiler takes your code written in C, removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.